lib/paper_trail_scrapbook/changes.rb

```diff
@@ -63,17 +71,40 @@ def find_value(key, value)
       return '*empty*' unless value
 
       begin
-        build_associations[key].find(value).to_s.to_s + "[#{value}]"
+        assoc_target(key).find(value).to_s.to_s + "[#{value}]"
```

Comment:
what's up with the double `to_s`? Seems odd

Reply:
Good question, but my recall is there is a reason for it.

Reply:
Yes - it is a case where if the to_s is bil then it figures it was not found even though it was found and the to_s is nil.